zulip/integrations/bridge_with_irc/requirements.txt

```diff
@@ -1 +1 @@
-irc==18.0
+irc==20.3.0
```

Comment:
Should this be `>= 20.3.0` or `~= 20.3`?

Reply:
I changed it to `~=20.3`.